Source/Orts.Simulation/Simulation/Timetables/TTTrain.cs

```diff
@@ -9566,7 +10014,8 @@ public override void CheckStationTask()
                             PresentPosition[1].Reverse(ValidRoute[0][PresentPosition[1].RouteListIndex].Direction, tempRoute, 0.0f, signalRef);
 
                             // reverse formation
-                            ReverseFormation(false);
+//                            ReverseFormation(false);
```

Comment:
Please remove commented out code, especially when it's been replaced by something new.

Reply:
Done

Reply:
Well, I would have if the system wasn't hiding the 'commit' button?